platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleRoot.java

```diff
@@ -56,6 +56,14 @@ public interface MerkleRoot extends MerkleInternal {
     @NonNull
     PlatformStateModifier getWritablePlatformState();
 
+    /**
+     * Gets the Roster state modifier.
+     *
+     * @return the Roster state modifier.
+     */
+    @NonNull
+    RosterStateModifier getRosterStateModifier();
```

Comment:
It makes sense that this method is named `getRosterStateModifier()` since it returns an instance of `RosterStateModifier`. However, there is an similar pre-existing method in this class with a different name:

`PlatformStateModifier getWritablePlatformState();`

I am not a fan of the mismatch in the return type and getter, however, the name of the method is very useful because it really jumps out as being writable. Maybe it's only clearer just to me. I think we should decide on the naming conventions and make it consistent. I lean toward making the names for states all start with `Readable` or `Writeable` to match the State API methods (`getWriteableStates()` and `getReadableStates()`) that we are shifting toward.

This can be worked out separately and with a different ticket.

Reply:
Sure. If there are further changes to make in this PR, I'll include this renaming in it.

Reply:
done